### ISSUE TITLE
Template doesn't update the logbook card

### ISSUE THREAD

**Issue by Chris-V** [thomasloven/lovelace-auto-entities#155]
I'm trying to build a logbook card with a Jinja2 template and this basically does nothing.

In fact, there are two issues when using auto-entities with a the logbook card and a template:

1. `entities` is mandatory. otherwise I get a red config error.
2. The template result does not update the card.

Here's a very simple reproduction:

```yaml
- type: custom:auto-entities
  card:
    type: logbook
    title: House feed
  entities:
    - light.office
  filter:
    template: "{{ ['light.bookshelf'] }}"
```

Running on HA 2020.12.

Thank you for the awesome work!

**Comment by Chris-V** [comment]
I looked at the issue a bit more and it seems to work "sometimes".

On a full refresh (Ctrl-R), I get this error in the console:
> auto-entities.js:1:12297 Uncaught TypeError: this.card.setConfig is not a function

However if I do a bunch of refresh (sometimes 2, sometimes 5, it varies) from the lovelace menu > refresh, then it will eventually work as expected.

**Comment by Chris-V** [comment]
Here's more!

When I leave the tab open, the logbook actually fetches events for all the entities returned by my template. It's just the old events that are not always displayed. So I digged a little bit in https://github.com/home-assistant/frontend/blob/dev/src/panels/lovelace/cards/hui-logbook-card.ts

My issue seems to be caused by two things:
1. The card requires non-empty entities to render, so I need an initial value
2. This then triggers a logbook fetch that is throttled to once every 10 seconds. The logbook card fetches historical data only once, hence the new entities only show up for future events.

I don't think auto-entities can do a lot to circumvent that. Perhaps delay the rendering of the underlying card until after the filtering and/or template have resolved?

**Comment by thomasloven** [comment]
Try release 1.9.0

**Comment by Chris-V** [comment]
The latest release appears to solve this. Thank you for taking care of this!

**Comment by ScottG489** [comment]
I have intermittent issues with auto-entites not displaying at all. It's particularly bad on Fully Kiosk, but I often see it on other devices as well. This describes it perfectly:
> However if I do a bunch of refresh (sometimes 2, sometimes 5, it varies) from the lovelace menu > refresh, then it will eventually work as expected.

And the error you pasted is exactly the one I'm having:
```
TypeError: this.card.setConfig is not a function
```
Searching this error on Google and this issue is only one of two results. 

I've also seen similar behavior with the switch-state card, but auto entities was slightly more reliable. I've seen this universally though, not specifically to logbook

**I'll possibly be posting my own issue once I gather more information**, but I thought I'd post something sooner here in case there's been an obvious regression that could be identified or if anyone has any other thoughts.

I hooked up the remote Webview debugger to Fully Kiosk in order to get this error:
```
Uncaught (in promise) TypeError: this.card.setConfig is not a function
    at Wt.update_card (auto-entities.js?hacstag=1677445841121:172:5063)
    at Wt.update_all (auto-entities.js?hacstag=1677445841121:172:3141)
```

**Comment by ildar170975** [comment]
@ScottG489
You should definitely create a new issue.
And first test w/o Kiosk.
